lib/resources/example_json/r4_examples_coverage.rb

```diff
@@ -187,7 +187,7 @@ module Resources
             'resourceType': 'Coverage',
             'id': 'PH-98371617-97701467',
             'meta': {
-              'versionId': '2',
+              'versionId': '2-0',
```

Comment:
Just to confirm, the versionId got updated to "2-0"?

Reply:
yup, @CBrashear can confirm, but this is just to show there is now a hyphen, {int}-{int} value rather than just an int

Reply:
The versionId for public healthcares will be in this format.